src/Orchard.Web/Modules/Orchard.OpenId/Module.txt

```diff
@@ -0,0 +1,38 @@
+﻿Name: OpenId Connect
+AntiForgery: enabled
+Author: Tha'er Al-Ajlouni, Avertra Corp. (http://www.avertra.com)
+Website: http://orchardproject.net
+Version: 1.0
+OrchardVersion: 1.10.1
+Description: Enables Orchard to authenticate users using OpenId
+Category: OpenId Providers
+Features:
+    Orchard.OpenId:
+        Description: Enables Orchard to authenticate users using OpenId
+		Category: Authentication
+		Name: OpenId
+    Orchard.OpenId.Facebook:
+        Description: Enables Orchard to authenticate users using their Facebook Accounts
+		Category: OpenId Providers
```

Comment:
I think the category of the providers should be "Authentication" just like the root feature.